pkg/ccl/logictestccl/testdata/logic_test/partitioning_enum

```diff
@@ -55,3 +55,29 @@ ALTER TABLE partitioned_table_3 PARTITION BY RANGE (place)
 
 statement ok
 SELECT * FROM crdb_internal.tables
+
+
+subtest drop_enum_partitioning_value
+
+statement ok
+drop table if exists tbl;
+drop type if exists t;
+set enable_drop_enum_value = true;
+create type t as enum('a', 'b', 'c');
+create table tbl (pk t PRIMARY KEY) PARTITION BY LIST (pk) (PARTITION "a" VALUES IN ('a'));
+alter partition "a" of table tbl configure zone using gc.ttlseconds = 1;
```

Comment:
I'm genuinely curious as to why this is even needed.

Reply:
I just cargo-culted this one.

Reply:
Removed